docs/sources/articles/networking.md

```diff
@@ -499,7 +505,8 @@ want to configure `eth0` via Router Advertisements you should set:
 ![](/article-img/ipv6_basic_host_config.svg)
 
 Every new container will get an IPv6 address from the defined subnet. Further
-a default route will be added via the gateway `fe80::1` on `eth0`:
+a default route will be added on `eth0` via the address specified by the daemon
```

Comment:
Indeed. Both mentions of 'eth0' refer to the interface of the container. Removed the second reference to "eth0" and added "in the container".
```
Every new container will get an IPv6 address from the defined subnet. Further
a default route will be added on `eth0` in the container via the address
specified by the daemon option `--default-gateway-v6` if present, otherwise
via `fe80::1`:
```